src/core/tests/frontend/paddle/test_models/gen_scripts/generate_elementwise_ops.py

```diff
@@ -193,5 +221,29 @@ def main():
     axis = 0
     elementwise_ops("4", data_x, data_y, axis, in_dtype)
 
+    # test for elementwise_floordiv, support int and int64
+    # paddle1.8 support axis = [0, x_last_dims]
+    # paddle2.x only support axis = -1
+    floordiv_support_dtype = ['int64', 'int32']
+    data_x = np.array([-2, 0, 4])
+    
+    data_y = np.array([1, 5, 2])
+    axis = -1
+    for dtype in floordiv_support_dtype:
+        elementwise_floordiv("elementwise_floordiv_" + dtype + "_1", 
+                        data_x.astype(dtype), data_y.astype(dtype), axis, dtype)
+
+    data_x = np.random.randint(1, 10, [2, 5, 3, 4])
+    data_y = np.random.randint(1, 5, [3, 4])
```

Comment:
Hi @taixiurong could add the test case with negative value, seems Paddle's implementation is aligned with Torch.

Reply:
> Hi @taixiurong could add the test case with negative value, seems Paddle's implementation is aligned with Torch.

1. yes, i review the paddle code , floor_div to call std::trunc(a/b), https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/phi/kernels/funcs/elementwise_functor.h#L573
but the paddle give a response：
<img width="905" alt="image" src="https://user-images.githubusercontent.com/6076976/191889356-a791f7f3-1f4e-4b23-aea7-20c3fd9c5013.png">

https://github.com/PaddlePaddle/Paddle/issues/46379
2. in this test case , i use a = -2, b = 1, is ok.  a = -4, b = 3, z = floor_div(a/b), z = -1。 so want to modiy this logic in openvino?

Reply:
@ceciliapeng2011 @meiyang-intel what's your opinion ?

Reply:
@OpenVINO-dev-contest hi  paddle fix this bug,  
doc:  https://github.com/PaddlePaddle/Paddle/pull/46419/files
code: https://github.com/PaddlePaddle/Paddle/pull/45051/files
i fix the code in openvino,  tese code like:
![image](https://user-images.githubusercontent.com/6076976/191932974-18496fe4-9285-4c98-b47e-85bff3778d55.png)

the tese result:
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/6076976/191933075-999178cf-a040-43a4-958f-79b611ae0161.png">

do  want push new code ?

Reply:
of course, you can try it

Reply:
update

Reply:
Can you test float to check if OpenVINO and Paddle can have the same result too? I am afraid they won't.

OpenVINO divide op has an attribute "pythondiv", but works with integer only. That's why the integer tests pass.
https://github.com/openvinotoolkit/openvino/blob/7bbd428e4814246677181ebffcac2379279367c3/src/core/reference/include/ngraph/runtime/reference/divide.hpp#L30
https://github.com/openvinotoolkit/openvino/blob/7bbd428e4814246677181ebffcac2379279367c3/src/core/include/openvino/op/divide.hpp#L29

Reply:
@jane-intel Do you have comments about the algorithm discrepancy of openvino divide and paddle floor_divide? Thanks.

Reply:
https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/phi/kernels/elementwise_kernel.cc#L144  paddle support int and int64, not support float.  
the paddle finally call return static_cast<T>(a / b).  
https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/phi/kernels/funcs/elementwise_functor.h#L573
in openvino if pythondiv is false , to call out[i] = arg0[i] / arg1[i];
https://github.com/openvinotoolkit/openvino/blob/7bbd428e4814246677181ebffcac2379279367c3/src/core/reference/include/ngraph/runtime/reference/divide.hpp#L48
the logic is same.